src/lib/core/option/_option-theme.scss

```diff
@@ -30,7 +30,7 @@
 
     // In multiple mode there is a checkbox to show that the option is selected.
     &.mat-selected:not(.mat-option-multiple) {
-      background: mat-color($background, hover);
+      background: rgba(mat-color($background, hover), 0.12);
```

Comment:
fyi `mat-color` should take opacity as a third argument, but it's the same either way